### PR TITLE
Add cosmological argument

### PR DESCRIPTION
**Pull request by wei-ma**
Please review this, @casey-collab

### CODE REVIEW

File: english/philosophy/cosmological_argument.md [hunk 1]
```diff
@@ -0,0 +1,4 @@
+In the morning, you recited cosmological argument:
```
Comment (casey-collab) [inline english/philosophy/cosmological_argument.md]: It looks like this file has been added to a directory that already contained a "page" of the story.  Please create a new directory for each new page to help keep the repository organized.  See the README of this repository for more information.

**Comment by casey-collab** [comment]
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

**Comment by wei-ma** [comment]
Please review this, @casey-collab

File: english/philosophy/cosmological_argument.md (hunk 1 above)
Comment (casey-collab) [inline english/philosophy/cosmological_argument.md]: It looks like this file has been added to a directory that already contained a "page" of the story.  Please create a new directory for each new page to help keep the repository organized.  See the README of this repository for more information.

Comment (casey-collab) [inline english/philosophy/cosmological_argument.md]: It looks like this page is not reachable from the rest of the story. That is, if a reader starts in `language.md` and keeps clicking links, they will never find this page. You can fix this by finding a good place in the existing story to add an option with a link that leads to this page.

**Comment by casey-collab** [comment]
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

**Comment by wei-ma** [comment]
Please review this, @casey-collab

File: english/philosophy/cosmological_argument.md (hunk 1 above)
Comment (casey-collab) [inline english/philosophy/cosmological_argument.md]: It looks like this file has been added to a directory that already contained a "page" of the story.  Please create a new directory for each new page to help keep the repository organized.  See the README of this repository for more information.

Comment (casey-collab) [inline english/philosophy/cosmological_argument.md]: It looks like this page is not reachable from the rest of the story. That is, if a reader starts in `language.md` and keeps clicking links, they will never find this page. You can fix this by finding a good place in the existing story to add an option with a link that leads to this page.

**Comment by casey-collab** [comment]
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

**Comment by wei-ma** [comment]
I add an option with a link that leads to this page.
Please review this, @casey-collab

File: english/philosophy/cosmological_argument.md (hunk 1 above)
Comment (casey-collab) [inline english/philosophy/cosmological_argument.md]: It looks like this file has been added to a directory that already contained a "page" of the story.  Please create a new directory for each new page to help keep the repository organized.  See the README of this repository for more information.

File: english/philosophy/questions.md [hunk 2]
```diff
@@ -1 +1,3 @@
-We have the questions. Now we need to find the answers. Look to the universe.
\ No newline at end of file
+We have the questions. Now we need to find the answers. Look to the universe.
+
+Maybe the Universe is created by [god](../../cosmological_argument/cosmological_argument.md)
```
Comment (casey-collab) [inline english/philosophy/questions.md]: It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.

File: english/philosophy/cosmological_argument.md (hunk 1 above)
Comment (casey-collab) [inline english/philosophy/cosmological_argument.md]: It looks like this page is not reachable from the rest of the story. That is, if a reader starts in `language.md` and keeps clicking links, they will never find this page. You can fix this by finding a good place in the existing story to add an option with a link that leads to this page.

**Comment by casey-collab** [comment]
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

**Comment by wei-ma** [comment]
I tried to fix the link, 
Please review this, @casey-collab

**Comment by casey-collab** [comment]
Looks great! :four_leaf_clover:

**Pull request merged**